ui/webpack.config.js

```diff
@@ -17,241 +17,243 @@ const eslintConfig = {
   baseConfig,
 };
 
-module.exports = {
-  mode: 'development',
-  devtool: 'cheap-module-eval-source-map',
-  target: 'web',
-  context: __dirname,
-  stats: {
-    children: false,
-    entrypoints: false,
-    modules: false,
-  },
-  node: {
-    Buffer: false,
-    fs: 'empty',
-    tls: 'empty',
-  },
-  output: {
-    path: `${__dirname}/build`,
-    publicPath: '/',
-    filename: 'assets/[name].js',
-    globalObject: 'this',
-  },
-  resolve: {
-    alias: {
-      'react-native': 'react-native-web',
-      'react-dom': '@hot-loader/react-dom',
+module.exports = env => {
+  return {
+    mode: env,
+    devtool: env === 'production' ? false : 'cheap-module-eval-source-map',
```

Comment:
Don't use `'cheap-module-eval-source-map'` in production builds.

Reply:
This is probably worth a comment in the code explaining why not.